Source/PeripheralManager.swift

```diff
@@ -54,6 +78,32 @@ class PeripheralManager: ManagerType {
 
     // MARK: Advertising
 
+    /// Starts peripheral advertising on subscription. It create inifinite observable
+    /// which emits only one next value, of enum type `StartAdvertisingResult`, just
```

Comment:
maybe change 'only one next value' to 'single next value' ? up to you @paweljaneczek